.github/workflows/notify.yaml.tmp

```diff
@@ -0,0 +1,15 @@
+# name: MS Teams Deploy Card
```

Comment:
Is this wf needed as it is all disabled?

Reply:
i wasnt sure. If the team agrees to remove it, I'll do it.

Reply:
lets remove the notify jobs everywhere until we have a plan :)